core/server/utils/zip-folder.js

```diff
@@ -0,0 +1,20 @@
+var archiver = require('archiver'),
+    fs = require('fs');
+
+module.exports = function zipFolder(folderToZip, destination, callback) {
+    var output = fs.createWriteStream(destination),
```

Comment:
https://github.com/archiverjs/node-archiver/issues/148
We can add this later if you want 👍